modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/util/PatternUtil.java

```diff
@@ -32,30 +30,18 @@ public class PatternUtil {
 	public static List<RedirectPatternEntry> parse(String[] patternStrings) {
 		List<RedirectPatternEntry> redirectPatternEntries = new ArrayList<>();
 
-		int limit = 2;
-
-		if (FeatureFlagManagerUtil.isEnabled("LPS-175850")) {
-			limit = 3;
-		}
+		int limit = 3;
 
 		for (String patternString : patternStrings) {
 			String[] parts = patternString.split("\\s+", 3);
 
 			if ((parts.length < limit) || parts[0].isEmpty() ||
-				parts[1].isEmpty()) {
+				parts[1].isEmpty() || parts[2].isEmpty()) {
 
 				continue;
 			}
 
-			String userAgent = RedirectConstants.USER_AGENT_ALL;
-
-			if (FeatureFlagManagerUtil.isEnabled("LPS-175850")) {
-				if (parts[2].isEmpty()) {
-					continue;
-				}
-
-				userAgent = parts[2];
-			}
+			String userAgent = parts[2];
```

Comment:
There's no need for declaring variables `limit` and `userAgent`, they can be inlined right?